example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -168,7 +170,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
-						DevelopmentTeam = 2N9YTZSQBW;
+						DevelopmentTeam = HB3FQU7HQM;
```

Comment:
can you remove the changes in this file? Also, is the entitlements file with the `keychain-access-groups` required?

Reply:
Will change team, keychain might be required will check that